scripting/get5.sp

```diff
@@ -230,6 +231,8 @@ public void OnPluginStart() {
         "Maximum number of pauses a team can use, 0=unlimited");
     g_MaxPauseTimeCvar = CreateConVar("get5_max_pause_time", "300",
         "Maximum number of time the game can spend paused by a team, 0=unlimited");
+    g_QuickRestartCvar = CreateConVar("get5_quick_restarts", "0",
```

Comment:
"quick restarts" isn't a good name for this. get5_use_lo3 would be much descriptive.